org.mixedrealitytoolkit.input/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.input",
-  "version": "3.0.1-development",
```

Comment:
Does this need to be incremented? I don't think we've shipped 3.0.1 of Input yet. (Same question from some of the other package version changes too)

Reply:
I was assuming these changes would go in after the next release... I can change these if needed